bonsai/sdk/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "bonsai-sdk"
 description = "Bonsai Software Development Kit"
-version = { workspace = true }
+version = "1.4.0-rc.1"
```

Comment:
shouldn't this be 2.0 too?

Reply:
We're moving to a new model where each component has its own version.